.babelrc

```diff
@@ -1,4 +1,13 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react"],
-  "plugins": ["babel-plugin-styled-components"]
+  "presets": [
+    ["@babel/preset-env", {
+      "useBuiltIns": "usage",
```

Comment:
How does this affect `imports`? Can you add a note in README as it's not the default? Please add that these two options are correlated (and how).